beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -33,7 +33,10 @@ func (r *RegularSync) validateBeaconAttestation(ctx context.Context, msg proto.M
 	// TODO(1332): Add blocks.VerifyAttestation before processing further.
 	// Discussion: https://github.com/ethereum/eth2.0-specs/issues/1332
 
-	att := msg.(*ethpb.Attestation)
+	att, ok := msg.(*ethpb.Attestation)
```

Comment:
no need to check for the type here, the handler before hand checks the message type so this is a redundant check.